src/main/kotlin/net/djvk/fireflyPlaidConnector2/sync/PolledSyncRunner.kt

```diff
@@ -157,22 +162,28 @@ class PolledSyncRunner(
                                 cursorMap[accessToken],
                                 plaidBatchSize
                             )
```

Comment:
I did get a bit carried away with my suggested changes, but I do think my suggestion here has merit.

Changing line 140 to 
```
accessTokenLoop@ for ((accessToken, accountIds) in accountAccessTokenSequence) {
```

and here to 
```
) ?: continue@accessTokenLoop
```

will be cleaner than adding 10 safe call operators.

Reply:
Aha, indeed, this is where I started with a `?let { it } ?: run { continue }` but of course continue cannot be used in a lamdba.  This is exactly the syntax I needed - good suggestion.

Reply:
Took this suggestion in both spots.